server/src/test/java/io/crate/integrationtests/Bug13380IntegrationTest.java

```diff
@@ -0,0 +1,433 @@
+/*
+ * Licensed to Crate.io GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.integrationtests;
+
+import static io.crate.testing.TestingHelpers.printedTable;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+import org.apache.lucene.tests.util.TestUtil;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.env.Environment;
+import org.elasticsearch.test.IntegTestCase;
+import org.junit.Test;
+
+@IntegTestCase.ClusterScope(numDataNodes = 0, numClientNodes = 0)
+public class Bug13380IntegrationTest extends IntegTestCase {
+
+    @Test
+    public void test_swap_table_partitioned_to_partitioned_1() throws Exception {
+        startUpNodeWithDataDir("/indices/data_home/cratedata-5.1.2-swap-table-bug-#13380.zip");
+        //create table m1.t1 (t boolean) partitioned by (t) with (refresh_interval = 300);
+        //create table m1.s1 (s char) partitioned by (s) with (refresh_interval = 400);
+        //insert into m1.t1 values (true), (false), (false);
+        //insert into m1.s1 values ('a'), ('a'), ('b');
+        //alter cluster swap table "m1"."t1" to "m1.s1";
```

Comment:
why are these commented out? are these the steps used to create the broken state? If so - add a comment telling that. Same further below.

Why are custom refresh intervals necessary for this?

Reply:
> are these the steps used to create the broken state? If so - add a comment telling that. Same further below.

Yes, I will add the comment.

> Why are custom refresh intervals necessary for this?

I just wanted to confirm settings were not affected.